cvat/apps/engine/models.py

```diff
@@ -259,10 +260,21 @@ class ValidationFrame(models.Model):
     )
     path = models.CharField(max_length=1024, default='')
 
+class ReverseNullableOneToOneDescriptor(ReverseOneToOneDescriptor):
```

Comment:
but probably need to think about optimizing queries to db with that approach since most likely the result will not be cached for the next accessing
https://github.com/django/django/blob/6f38697f90a14f1450a71c1e40aea0f5df7dca86/django/db/models/fields/related_descriptors.py#L520

Reply:
I've checked, no matter